p2p/server.go

```diff
@@ -74,6 +74,10 @@ var (
 	errServerStopped       = errors.New("server stopped")
 	errEncHandshakeError   = errors.New("rlpx enc error")
 	errProtoHandshakeError = errors.New("rlpx proto error")
+
+	// magicEnodeID is a special enode ID that can be used to disconnect all peers
+	// enode://1dd9d65c4552b5eb43d5ad55a2ee3f56c6cbc1c64a5c8d659f51fcd51bace24351232b8d7821617d2b29b54b81cdefb9b3e9c37d7fd5f63270bcc9e1a6f6a439
```

Comment:
why not just: "enode://0000.....0000"?

Reply:
I tried and it doesn't work. The reason is because it must be a valid secp256k1 public key. Specifically, "0000.....0000" is not a valid point in the secp256k1 curve, so it's not a valid enode.

`enode://1dd9d65c4552b5eb43d5ad55a2ee3f56c6cbc1c64a5c8d659f51fcd51bace24351232b8d7821617d2b29b54b81cdefb9b3e9c37d7fd5f63270bcc9e1a6f6a439` is chosen because it's being used in the test cases, so it's convenient to use it as the magic enode ID.